web/src/components/hero.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BackgroundImage from 'gatsby-background-image';
+
+export default function Hero({ image }) {
+    const {
+        childImageSharp: { fluid },
+    } = image;
+
+    return (
+        <BackgroundImage
+            tag="section"
```

Comment:
I think `tag` needs to be capitalized. Check the [docs](https://www.gatsbyjs.com/plugins/gatsby-background-image/#how-to-use). (or check if the generated html is a `section` element)